tests/core/remodel-kernel/task.yaml

```diff
@@ -7,8 +7,6 @@ details: |
     the new kernel snap. Finally check it is possible to remodel
     back to the initial model.
 
-# FIXME: add core18 test as well
-# TODO:UC20: enable for UC20
 systems: [ubuntu-core-16-64]
```

Comment:
Why are we removing the TODO but not changing the `systems`?

Reply:
This is because we are testing different kernel remodeling in nested tests, and this one cannot be updated to work in uc20+ without a big refactor.

Reply:
can we rename the test to specifically indicate this one is for uc16 and not generic. The test name should probably be `uc16-remodel-kernel` and not `remodel-kernel`

Reply:
Otherwise it will give a false sense of security when looking at the test overview, we want to be explicit we are not testing this for other systems, and that we in fact need a new test for this

Reply:
updated